server.js

```diff
@@ -13,12 +16,80 @@ app.get('/fruits', (req, res) => {
   res.json(fruits)
 });
 
+//GET
 app.get('/fruits/:id', (req, res) => {
-  res.json({})
+  console.log(req.params)
+
+  //fruiFound is gonna be an object
+  let fruitFound = fruits.find(fruit => fruit.id == req.params.id )
+
+  if(!fruitFound){
+    return res.json({message: `we don't have any ${req.params.id}`})
+  }else{
+    return res.json({message:`we have the ${fruitFound.name} and its color is ${fruitFound.color} ?`})}
 })
 
+//POST
+app.post('/newfruit', (req, res) => {
```

Comment:
A minor: The name of the route we typically leave the same for all REST routes.
So we would use:
get('/fruit')
post('/fruit')
patch('/fruit/:id")
and so on...

That way it is easier for the frontend if we follow that standard